composer.json

```diff
@@ -23,7 +23,7 @@
         "drupal/drupal-extension": "^4.0.0@alpha",
         "openeuropa/code-review": "^0.2",
         "openeuropa/task-runner": "^0.5",
-        "openeuropa/oe_theme": "^0.2",
+        "openeuropa/oe_theme": "dev-OPENEUROPA-615",
```

Comment:
Why does this require a dependency change on the theme? The scope of this ticket is to set the language of the splash page, this should work regardless of the theme being used.

This should even work if we are using Bartik as a theme.